hugo/content/3-queries/operators.md

```diff
@@ -17,6 +17,12 @@ They work on any field type, and the right-hand value must be the same type as t
 Free functions are also provided in the `mangrove` namespace that can be used in lieu of the overloaded
 operators.
 
+{{% notice tip %}}
+When working with array fields, one can also compare them to values that are the type of the array's elements.
+This provides the same functionality as using the `$elemMatch` operator.
```

Comment:
There is a subtle difference here between elem match and the raw operators. For example, if I have some documents like this:
```js
{
  a: [ { x: 4, y: 1 }, { x: 0, y: 4 } ]
}
{
  a: [ { x: 0, y: 1 }, { x: 4, y: 4 } ]
}
```

Then this can happen:
```
> db.test.find({ 'a.x': 0, 'a.y': 1 })
{ "_id" : ObjectId("579f984e454ad917bc9f6025"), "a" : [ { "x" : 4, "y" : 1 }, { "x" : 0, "y" : 4 } ] }
{ "_id" : ObjectId("579f984e454ad917bc9f6026"), "a" : [ { "x" : 0, "y" : 1 }, { "x" : 4, "y" : 4 } ] }

> db.test.find({ 'a': { '$elemMatch': { 'x': 0, 'y': 1 } } })
{ "_id" : ObjectId("579f984e454ad917bc9f6026"), "a" : [ { "x" : 0, "y" : 1 }, { "x" : 4, "y" : 4 } ] }
```

elemMatch has to match with a single element, whereas just comparing to a scalar is done individually and will return documents where any element matches.